gym/lib/gym/code_signing_mapping.rb

```diff
@@ -104,6 +104,21 @@ def test_target?(build_settings)
       return (!build_settings["TEST_TARGET_NAME"].nil? || !build_settings["TEST_HOST"].nil?)
     end
 
+    def same_platform?(sdkroot)
+      destination = Gym.config[:destination].dup
+      destination.slice! "generic/platform="
```

Comment:
style nit, we like to use ()'s: 
`destination.slice! "generic/platform="` ->`destination.slice!("generic/platform=")`